### DISCUSSION TITLE
看一看中国共产党党员组成 | /project/dangyuan/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#625] [Announcements]
https://yuanfan.rbind.io/project/dangyuan/

**Comment by botsp** [comment]
2021，正是以前如火如荼的那一年了，想必是很多先进分子立功受奖了。

**Reply by botsp** [reply]
*疫情

**Reply by earfanfan** [reply]
如果说立功受奖这方面的话，其实2021年是建党百年。按照我们国家千百年的文化传统，会倾向于纪念形成一个整数的年份，比如老人家愿意庆祝七十大寿、八十大寿。建党百年的时候，同年颁发表彰的数量也是很多的。

|年份|表彰先进基层党组织（万）|表彰优秀共产党员（万）|表彰优秀党务工作者（万）|光荣在党50年纪念章颁发（万）|
|:--:|:--:|:--:|:--:|:--:|
|2023|13.8|69.3|19.9|79.5|
|2022|9.8|55.7|14.1|99.5|
|2021|**44.4**|**153.1**|**55.5**|**729.3**|
|2020|||||
|2019|19.9|91|23.2||
|2018|19|96.2|21.7||
|2017|17|96.6|16.8||
|2016|30.4|140|28.1||
|2015|15.8|91|15.7||
|2014|16.3|90.8|16.2||
|2013|21|110|19.6||
|2012|36.2|147.3|24.6||

**Reply by botsp** [reply]
有道理；；；光荣在党50年纪念章颁发（万)"，这个是指党龄50年吗，为啥在2021有729.3万这么多，其它年份少很多，1971年发生了啥

**Reply by earfanfan** [reply]
也可能是补发以往年份的。

**Reply by botsp** [reply]
往前补发五年的，平均90*5=450万，这样2021还是发下去了279万，好多，哈哈

**Reply by earfanfan** [reply]
也许不止补发五年吧，你看看下面这个数据，也是我从中国共产党党内统计公报里整理的。
|年份|新中国成立前入党|新中国成立后至“文革”前入党|“文革”期间入党|新中国成立后至党的十一届三中全会前入党|
|:--:|:--:|:--:|:--:|:--:|
|2023|6.8|||1286.3|
|2022|9.6|||1360.9|
|2021|11.9|||1417.5|
|2019|17.4|||1550.9|
|2018|20.5|||1616|
|2017|||||
|2016|29.3|538.4|1039.5|1823.8|
|2015|34.3|579.3|1070.3|1903.4|
|2014|38.9|609.6|1092.5||
|2013|43.9|642.8|1117.1||

**Reply by botsp** [reply]
明白了，还是特殊历史时期的问题了...

**Comment by HushWay** [comment]
好奇，没有看到直接可以下载的数据文件，数据是通过爬虫之类的方式获取的么？

**Reply by earfanfan** [reply]
从官网上整理好的数据文件我也上传了，在这里<https://github.com/earfanfan/yuanfan/tree/main/content/project/2024-07-07-dangyuan/data>。

哈哈，其实数据就是我手动复制粘贴整理的。

**Reply by HushWay** [reply]
原来如此，你好有耐心哦。我就很懒，被略繁琐的思想汇报劝退没有入党，哈哈

**Reply by earfanfan** [reply]
其实我之前写的关于自杀率、出生人口性别比、耕地面积等数据全都是手动复制粘贴整理的，倒不是人有耐心，只是内心有动力去做罢了。

话说回来，我上大学的时候连成为入党积极分子的资格都没有呢，好像那时候入党的名额每年都特别少，不过我们宿舍4个人有2个人入党了。

**Reply by HushWay** [reply]
自杀率那篇其实我就好奇啦。根据你的数据马后炮一下，我最该入党的年份是2021年，天时（建党百年）地利（比较闲）人和（学生），可惜错过了，哈哈

**Comment by XiangyunHuang** [comment]
这是我在逛共产党历史博物馆时获得全国党员人数

|年份 | 全国党员人数|
|--|--|
1923年| 420人
1925年| 994人
1927年| 57967人
1928年| 没有准确数字
1969年| 2200万
1973年| 2800万
1977年| 3500多万
1982年| 3900多万
1987年| 4600多万
1997年| 5800多万
2002年| 6600多万
2007年| 7300多万
2012年| 8260多万

从拍的照片里获得的数字，到了文革，我才想起来要把历次大会（全国人大、中央全会）的人数拍下来，所以文革前缺了很多。

另外，我有个疑问，这个党员人数发展和全国总人口是否有密切关系，可否量化？

**Reply by XiangyunHuang** [reply]
这是逐年的人数变化

![IMG_7327-min](https://github.com/user-attachments/assets/d26b5c67-328b-4d0a-939e-7ea509251679)

这张照片统计党员人数的时间与上表不同，所以人数也不同

**Reply by earfanfan** [reply]
>到了文革，我才想起来要把历次大会（全国人大、中央全会）的人数拍下来，所以文革前缺了很多。

你的意思是说那个博物馆里居然本来就有，但是你没拍，那我猴年马月去北京的话一定要去瞅瞅。

>这个党员人数发展和全国总人口是否有密切关系，可否量化？

为了避免误解，我先说说自己对这个问题的理解。首先，相关关系不等于因果关系，这你肯定熟烂于心，就好像我上班带伞的时候，我的同事也会带伞，然而原因都只是天气预报说会下雨，所以我确实没有理解为何你会想到看党员人数和全国人口的关系。其次，我国有好几次整风整党，这个变量对党员人数的变化可能是影响很大的，但发展党员方面的政策因素应该不全是公开的。

**Reply by XiangyunHuang** [reply]
值得注意，可能由于政治风波，1990 年至2002年的党员人数和党组织数量图表是没有的，2003-2012 的图表如下

![IMG_7335-min](https://github.com/user-attachments/assets/1da55096-8e32-427a-ae49-49bded51b5c8)

**Reply by XiangyunHuang** [reply]
关于第一个问题，不一定全都有，就像展示党员人数和党组织数量图表这个信息类似，原因你懂的，必须是积极昂扬向上的图表。

关于第二个问题，我最近看了一点点萧公权的《中国乡村》，里面反复提到十进制，就是统治阶级和被统治阶级的人数比例，宝塔型的官僚阶级的组织逐级比例。再一般地想，15 亿人，15 个人来管理，肯定不够，通过官僚阶级，7-8 个层级放大，就是 1 亿 5000 万。

**Reply by XiangyunHuang** [reply]
如果有近 70 年的逐年数据（党员和全国人口数据），按照领导人任期划分，做一个简单的线性回归，是否可以观察出组织效率变化？我不知道我这个想法是否有合理，当然可以就某些拐点配合特殊事件一起解释。

**Reply by earfanfan** [reply]
额，实不相瞒，我想吐槽，但是要干活了，俺下班回去来叨叨。

**Reply by earfanfan** [reply]
我把之前整理的数据和湘云你提供的照片中的数据合在了一起，下面是党员数量。ps国家统计局网站有1949到2023年的人口数。

|年份|年末总人口(万人)|中国共产党党员总数（万人）|
|:--:|:--:|:--:|
|2023|140967|9918.5|
|2022|141175|9804.1|
|2021|141260|9671.2|
|2020|141212|9191.4|
|2019|141008|9059.4|
|2018|140541|8956.4|
|2017|140011|8944.7|
|2016|139232|8875.8|
|2015|138326|8779.3|
|2014|137646|8668.6|
|2013|136726|8512.7|
|2012|135922|8513|
|2011|134916|8260|
|2010|134091|8027|
|2009|133450|7800|
|2008|132802|7600|
|2007|132129|7415|
|2006|131448|7240|
|2005|130756|7080|
|2004|129988|6960|
|2003|129227|6820|



|年份|年末总人口(万人)|中国共产党党员总数（万人）|
|:--:|:--:|:--:|
|1989|112704|4950|
|1988|111026|4900|
|1987|109300|4780|
|1986|107507|4600|
|1985|105851|4400|
|1984|104357|4200|
|1983|103008|4100|
|1982|101654|4000|
|1981|100072|3970|
|1980|98705|3900|
|1979|97542|3800|
|1978|96259|3700|
|1977|94974|3600|
|1973|89211|2800|
|1969|80671|2200|

**Reply by earfanfan** [reply]
这是党内统计公报的党组织数量，不过不知道照片中的基层党组织是什么范围。

|年份|党的各级地方委员会|省（区、市）委|市（州）委|县（市、区、旗）委|城市街道|乡镇|社区（居委会）|行政村|机关基层党组织（万个）|事业单位基层党组织（万个）|企业基层党组织（万个）|公有制企业基层党组织（万个）|非公有制企业法人单位基层党组织（万个）|社会组织基层党组织（万个）|
|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
|2023|3199|31|397|2771|9125|29620|119437|488959|77.1|99.7|160|||18.3|
|2022|3198|31|397|2770|9062|29619|116831|490041|75.6|97.7|157.1|||17.9|
|2021|3198|31|397|2770|9034|29649|114065|491129|74.5|94.9|153.2|||17.1|
|2019|3202|31|397|2774|8636|31062|105257|533824|72.3|91.3|147.7|||14.2|
|2018|3208|31|398|2779|8561|31610|102555|545189|23|51.5|176.6|18.1|158.5|26.5|
|2017|3208|31|397|2780|8439|31726|100602|547152|23.2|51.6|206.2|18.5|187.7|30.3|
|2016|3207|31|396|2780|8138|31819|97911|550636|23.2|51.5||18.9|185.5|28.9|
|2015|3206|31|391|2784|7828|32341|94424|571544|23.3|51.1||19.2|160.2|20.4|
|2014|3218|31|397|2790|7565|32753|92581|577273|23.6|50.6||19.4|157.9|18.4|
|2013|3219|31|395|2793|7448|32972|90443|583975|23.5|50.8||19.5|162.7|11.5|
|2012|||||7245|33000|87000|588000|||||147.5|7.98|

**Reply by XiangyunHuang** [reply]
不着急哈，我只是想到一个分析点 --- 组织效率。也许合适的数据并不是我说的那个，你也可以找找已有的材料，我觉得分析组织效率，肯定有人做过了。

**Reply by earfanfan** [reply]
湘云呐，我想我一定没有理解你所表达的真正意思。不过我得先写出我的思考，这样的话，如果我真的误解了你，你也能看出来我误解的是撒。

首先，我确实没明白你想到的分析点——组织效率——是落在哪里。如果用共产党和组织效率这两个词一起搜索，有搜索出一些内容，但并不相关。如果用组织效率这个词单独搜索，可以搜到这是一个管理学方面的专业术语。讲真，这里我想起来很久以前的一件小事，记不清是在你的哪篇博客里，你把国家和经营这两个词语写到一起，当时我有去留言杠一把，记得我应该说的是企业经营的“经营”并不能直接迁移应用到治理国家的场景上。

其次，我搜了一下你提到的李公权写的《中国乡村》这本书，看简介写的是19世纪清朝统治下中国乡村的政治体系，我没有看过这本书，但在其他书里看到过一些相近的，感觉和建国后中国乡村经历过的种种变迁应该相差很大。你提到的宝塔型的官僚组织，这个其实也和我看过的“条条块块”不太一样，以前的乡村应该是皇权不过县，县以下由乡绅、宗族治理，科举制度取消后，士绅集团也分化，一部分向城市迁移，剩下滞留在乡村的劣化成为土豪劣绅，其治理的品质自然也急剧下降，甚至会直接演变成搅乱地方社会秩序的重大因素。不过我也不知道明朝时期的里甲制度到了清朝会变成什么样子，之前看过的一本书提到8090年代农村的小学都是村户和政府共建，而这个特色应该是清朝时期建立的，所以清朝时期的乡里制度可能也比之前的朝代在制度上有些进步。

说回总人口数和党员数这两个指标，人口数在不同时期增长速度不一样，而党员数也是，可是两者受到的影响因素却相差很远，像党员数从2007年到2012年每年净增200万，可是2013年却又比2012年还少。另外，党员其实并不能等同于组织者、管理者的角色，也并不是站在金字塔更上层的位置，发展党员的政策是向着年轻化、专业化，但是每年仍然会新增工人、农民。

虽然我不懂组织效率，可是也可以尝试思考组织和效率，政府办事效率也分常态时和动员时，比如疫情时期，动员效率是变得越来越快的，由于经济上存在区域间、城乡之间的巨大差异，所以这类差异可能也会传导到组织的办事效率上。
